modules/templates/helper.go

```diff
@@ -620,6 +622,17 @@ func Str2html(raw string) template.HTML {
 	return template.HTML(markup.Sanitize(raw))
 }
 
+// Markdown2html render Markdown text to sanitized HTML
+func Markdown2html(raw string) template.HTML {
```

Comment:
Can you rename this to `RenderMarkdown`? Would be more fitting.

Reply:
Or even more generic `RenderMarkup` with a argument string `markdown` for the type.

Reply:
I am not sure if that would work with a template or how to use it in template if it has 2 parameters

Reply:
Would a pipe like this `{{.Data | RenderMarkup "markdown"}}` work?

Reply:
I think it should just be `{{RenderMarkup "markdown" .Data}}` and the function than has `string, string` arguments.

Reply:
Ok, I made the changes, but had to use `Type` instead of `type`, not sure why.

Reply:
ah, I think that's because `type` is a reserved keyword in golang.

Reply:
I would go with `markupType` name for the argument. Uppercase variables in golang are usually meant for exported functions only.

Reply:
Ok, I will commit that change in a few mins, after current build finishes, but consider anything I did today as untested as I am away from proper environment and it seems there is no possibility to download artifacts from drone (or whatever CI it is)

Reply:
Should we really want to support other format types in the description?

Reply:
On hindsight, it probably doesn't makes sense to support more unless we can detect what the user has entered (e.g. via heuristics).